Source/WebInspectorUI/UserInterface/Views/AccessibilityTreeContentView.js

```diff
@@ -0,0 +1,331 @@
+            WI.domManager.highlightDOMNodeList([selectedDOMNode]);
+        }
+
+        this.dispatchEventToListeners(WI.ContentView.Event.SelectionPathComponentsDidChange);
+    }
+
+    _handlePathComponentSelected(event)
+    {
+        if (!event.data.pathComponent)
+            return;
+
+        console.assert(event.data.pathComponent instanceof WI.DOMTreeElementPathComponent);
+        console.assert(event.data.pathComponent.domTreeElement instanceof WI.DOMTreeElement);
+
+    }
+
+    _domNodeChanged(event)
+    {
+        var selectedDOMNode = this._domTreeOutline.selectedDOMNode();
+        if (selectedDOMNode !== event.data.node)
+            return;
+
+        this.dispatchEventToListeners(WI.ContentView.Event.SelectionPathComponentsDidChange);
+    }
+
+    _mouseWasClicked(event)
+    {
+        var anchorElement = event.target.closest("a");
+        if (!anchorElement || !anchorElement.href)
+            return;
+
+        // Prevent the link from navigating, since we don't do any navigation by following links normally.
+        event.preventDefault();
+        event.stopPropagation();
+
+        if (WI.isBeingEdited(anchorElement)) {
+            // Don't follow the link when it is being edited.
+            return;
+        }
+
+        // Cancel any pending link navigation.
+        if (this._followLinkTimeoutIdentifier) {
+            clearTimeout(this._followLinkTimeoutIdentifier);
+            delete this._followLinkTimeoutIdentifier;
+        }
+
+        // If this is a double-click (or multiple-click), return early.
+        if (event.detail > 1)
+            return;
+
+        function followLink()
+        {
+            // Since followLink is delayed, the call to WI.openURL can't look at window.event
+            // to see if the command key is down like it normally would. So we need to do that check
+            // before calling WI.openURL.
+            const options = {
+                alwaysOpenExternally: event?.metaKey ?? false,
+                frame: this._frame,
+                lineNumber: anchorElement.lineNumber,
+                ignoreNetworkTab: true,
+                ignoreSearchTab: true,
+            };
+            WI.openURL(anchorElement.href, options);
+        }
+
+        // Start a timeout since this is a single click, if the timeout is canceled before it fires,
+        // then a double-click happened or another link was clicked.
+        // FIXME: The duration might be longer or shorter than the user's configured double click speed.
+        this._followLinkTimeoutIdentifier = setTimeout(followLink.bind(this), 333);
+    }
+
+    _updateDOMTreeDeemphasizesNodesThatAreNotRendered()
+    {
+        this.element.classList.toggle("deemphasize-unrendered", WI.settings.domTreeDeemphasizesNodesThatAreNotRendered.value);
+    }
+
+    _handleDOMTreeDeemphasizesNodesThatAreNotRenderedChanged(event)
+    {
+        this._updateDOMTreeDeemphasizesNodesThatAreNotRendered()
```

Comment:
I think you can update the element from the listener rather than making another function.